### PR TITLE
Home

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
home page and singIn, signUp page

### CODE REVIEW

File: assets/images/R&B _ SignIn_files/style.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+}
+.image{
+    max-width: 100%;
+    object-fit: cover;
+}
+
+.product-categories{
+    display: flex;
+    align-items: center;
+    flex-direction: column;
```
Comment (ritikaawasthy) [inline assets/images/R&B _ SignIn_files/style.css:118]: you can use flex-flow here

File: assets/images/R&B _ SignIn_files/style.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+}    
+.store-input-box:focus-visible {
+    display: inline-block;
+    border: none;
+    outline: none;
+}    
+
+.btn{ 
+    border: none;
+    border-radius: 0.3rem;
```
Comment (ritikaawasthy) [inline assets/images/R&B _ SignIn_files/style.css:94]: you can use px here for border-radius

**Review by ritikaawasthy — APPROVED** [review]
Code looks clean

File: app.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "assets\\images\\Anime-silder-1920-1920x450.webp",
+    "assets\\images\\rick-and-morty-slider-1-1920-1920x450.webp",
+    "assets\\images\\harry-potter-battle-image@0,5x.webp"];
+
+    let i=0;
+    setInterval(()=>{
+        carousel.src=`${arrayOfImages[i]}`;
+        i+=1;
+        i>2?i=0:i=i;},5000);
+
```
Comment (manojsarna) [inline app.js:13]: looks good 🤔 😉

File: assets/images/R&B _ SignIn_files/style.css [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+    align-self: center;
+    margin-right: 1rem;
+}
+a{
+    text-decoration: none;
+    color: var(--color-white);
+}
+a:visited{
+    color: var(--color-white);
+}
```
Comment (manojsarna) [inline assets/images/R&B _ SignIn_files/style.css:56]: looks good!!

File: assets/images/R&B _ SignIn_files/style.css [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    border: none;
+    outline: none;
+}    
+
+.btn{ 
+    border: none;
+    border-radius: 0.3rem;
+    padding: 0.5rem 1.5rem;
+    cursor: pointer;
+}
```
Comment (manojsarna) [inline assets/images/R&B _ SignIn_files/style.css:97]: can use px values if vallue is less tahn 10 px!

File: assets/images/R&B _ SignIn_files/style.css [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 2rem;
+}
+.card{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-white);
```
Comment (manojsarna) [inline assets/images/R&B _ SignIn_files/style.css:138]: looks good!!

File: signIn and SignUp/signIn.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css">
+</head>
+<body>
+    <nav class="nav-container">
+        <a href=".\..\..\index.html">
+            <p class="nav-title">R<span class="nav-title-short">&</span>B <span class="nav-title-short">Store</span>
+            </p>
+        </a>
+
+        <div class="input-box">
```
Comment (manojsarna) [inline signIn and SignUp/signIn.html:19]: looks good!!

File: signIn and SignUp/signUp.css [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+                         "confpass -"
+                         "btn1 btn2";
+
+    grid-template-columns: 1fr 1fr ;
+    grid-template-rows: 0fr 0fr 0fr 0fr 0fr 0fr;
+    column-gap: 1rem;
+    row-gap: 9px;
+}
+
+.form-label{
```
Comment (manojsarna) [inline signIn and SignUp/signUp.css:17]: can use repeat in css grids

File: signIn and SignUp/signUp.css [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+#form-btn3{
+    grid-area: btn1;
+}
+#form-btn4{
+    grid-area: btn2;
+    text-align: center;
+    font-size: 13.33px;
+}
+
```
Comment (manojsarna) [inline signIn and SignUp/signUp.css:66]: looks gooD!!

File: signIn and SignUp/signin.css [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+.form-container{
+    display: flex;
+    place-content: center;
+    align-items: center;
+    height: 90vh;
+}
+
+.form-subcontainer{
+    padding: 2rem 4rem 4rem 4rem;
```
Comment (manojsarna) [inline signIn and SignUp/signin.css:49]: looks good!! , but to much css , do refactor the css code in future

File: style.css [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-white);
+    justify-content: center;
+    margin-right: 1rem;
+}
+
+.card-align-item-start{
+    align-items: start;
```
Comment (manojsarna) [inline style.css:155]: do check the css refactoring!!

**Review by manojsarna — APPROVED** [review]
Overall looks good

you can merge it!! :)

**Pull request merged**